doc/src/python.rst

```diff
@@ -8,14 +8,24 @@ Syntax
 
 .. parsed-literal::
 
-   python func keyword args ...
+   python function keyword args ...
 
-* func = name of Python function
-* one or more keyword/args pairs must be appended
+* function = *source* or name of Python function
```

Comment:
the left side of the = should not be "function" b/c it is confusing, esp in the next sentence
maybe it should be "mode"